robaka:
  hardware_interface:
     loop_hz: 50 # hz
     joints:
       - left_wheel
       - right_wheel
